app/routes/settings/exports.js

```diff
@@ -0,0 +1,25 @@
+import Route from 'ember-route';
+import get from 'ember-metal/get';
+import set from 'ember-metal/set';
+import { task } from 'ember-concurrency';
+import Pagination from 'client/mixins/pagination';
+
+export default Route.extend(Pagination, {
+  modelTask: task(function* () {
+    const results = yield get(this, 'store').findAll('linked-account', {
+      include: 'libraryEntryLogs.media'
+    });
+    const controller = this.controllerFor(get(this, 'routeName'));
+    set(controller, 'taskValue', results);
+  }),
+
+  model() {
+    return { taskInstance: get(this, 'modelTask').perform() };
+  },
+
+  actions: {
+    removeExport(exporter) {
+      exporter.destroyRecord();
+    }
+  }
+});
```

Comment:
Take a look at the imports route. The pagination code has changed and this code will need to be updated to support that.